### ISSUE TITLE
gulp dist runs tests but ignores the result

### ISSUE THREAD

**Issue by coltonw** [grommet/grommet-toolbox#8]
I would expect when the tests fail for the build to stop and fail, but instead it just keeps on running and considers the gulp build a success. Seems like a bug to me but I am not sure I can get to it right away like my other 2 pull requests.

**Comment by alansouzati** [comment]
Let me investigate this

**Comment by alansouzati** [comment]
It turns out that `gulp-tape` project does not support fail on error option. See this: https://github.com/yuanqing/gulp-tape/issues/15.

I was able to write a pull request to add that option, hopefully it will be accepted soon. As soon as it is accepted, we can do the following:

```
          gulp.src(options.testPaths)
            .pipe(envs)
            .pipe(tape({
              reporter: tapSpec(),
              failOnError: true
            }))
            .on('error', (error) => {
              gutil.log(error.message);
              process.exit(1);
            })
            .pipe(envs.reset)
            .pipe(istanbul.writeReports({
              dir: './coverage',
              reporters: ['lcov']
            })).on('end', () => {
              console.log('Test coverage report available at coverage/lcov-report/index.html');
              done();
            });
```

**Comment by alansouzati** [comment]
pending on https://github.com/yuanqing/gulp-tape/pull/18

**Comment by alansouzati** [comment]
Fixed 0.1.22 released ;)
